ayekan/exporters/snmp-exporter/configmap.yaml

```diff
@@ -7,14 +7,38 @@ metadata:
 data:
   snmp.yaml: |
     auths:
+      lsst_v1:
```

Comment:
If I understand the config correctly, only rubin_v2 is being used for polling. Is that correct?  If so, do we need to configure multiple community strings or was this intended to be an example?

Reply:
there is a single device that uses that authentication AFAIK. https://github.com/lsst-it/k8s-cookbook/pull/262/files/17566cfdbaa870c3c15af700dba47ce82c2bd2e5#diff-f44e3a33dbd81b8ea4f6fe7f2c4b9e877c988aa1265a4c24949ec4e2a5e3489eR114

Reply:
Doh, I'd missed that one.  It makes sense then.